src/Styleguide/Components/__stories__/AuctionTimer.story.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import { storiesOf } from "storybook/storiesOf"
+
+import { ContextProvider } from "Artsy/SystemContext"
+import { AuctionTimerQueryRenderer } from "Styleguide/Components/AuctionTimer"
+import { Section } from "Styleguide/Utils"
+
+storiesOf("Styleguide/Components", module).add("AuctionTimer", () => {
```

Comment:
These stories are going to just be closed in a few days (+ we have the regular `Timer` story), so I dunno how useful this really is. Can leave it for now.